main.cpp

```diff
@@ -0,0 +1,124 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+class Country{
+	public:
```

Comment:
Better use struct if you don't have any private methods

Reply:
done